azimuth/types/tag.py

```diff
@@ -147,10 +149,11 @@ class Config:
                 "data_actions": {
                     1: {
                         "relabel": True,
-                        "consider_new_class": True,
-                        "remove": True,
-                        "augment_with_similar": True,
-                        "investigate": True,
+                        "add_new_class": False,
+                        "merge_classes": False,
+                        "remove": False,
+                        "augment_with_similar": False,
```

Comment:
Tiny detail, but I notice that in most placer, you reordered `augment_with_similar` before those, so might as well do it here to for consistency:
```suggestion
                        "augment_with_similar": False,
                        "add_new_class": False,
                        "merge_classes": False,
                        "remove": False,
```